_data/team.yml

```diff
@@ -116,6 +116,13 @@ people:
   position: Bioinformatics Specialist, UHN
   team: implementation  
   
+  - name: Jessica Yang
+  lastname: Yang
```

Comment:
For YAML, indentation matters - please add spaces for "lastname", "firstname", etc to line up with "name".  Once that's done it should work!

Reply:
ok i've just fixed the indentation, please let me know if it still doesnt work!